price-feeder/oracle/oracle.go

```diff
@@ -69,6 +73,15 @@ type Oracle struct {
 	mtx             sync.RWMutex
 	lastPriceSyncTS time.Time
 	prices          map[string]sdk.Dec
+
+	oracleOnChain OnChainData
+}
+
+// OnChainData used to cache data that can be updated
+// at each cacheOnChainBlockQuantity amount of blocks
+type OnChainData struct {
```

Comment:
Can we move this struct, associated functions, and variables to another small file? Trying to keep the `oracle.go` file as small as we can

Reply:
sure thing